src/commands/synthetics/utils.ts

```diff
@@ -455,16 +455,62 @@ export const getReporter = (reporters: Reporter[]): MainReporter => ({
   },
 })
 
+const getTest = async (api: APIHelper, {id, suite}: TriggerConfig): Promise<{test: Test} | {errorMessage: string}> => {
+  try {
+    const test = {
+      ...(await api.getTest(id)),
+      suite,
+    }
+
+    return {test}
+  } catch (error) {
+    if (isNotFoundError(error)) {
+      const errorMessage = formatBackendErrors(error)
+
+      return {errorMessage: `[${chalk.bold.dim(id)}] ${chalk.yellow.bold('Test not found')}: ${errorMessage}`}
+    }
+
+    throw error
+  }
+}
+
+const getTestAndOverrideConfig = async (
+  api: APIHelper,
+  {config, id, suite}: TriggerConfig,
+  reporter: MainReporter,
+  summary: Summary
+) => {
+  id = PUBLIC_ID_REGEX.test(id) ? id : id.substr(id.lastIndexOf('/') + 1)
+
+  const getTestResult = await getTest(api, {config, id, suite})
```

Comment:
Kind of nit? I know you meant as "result of getTest" but the name could suggest that `getTestResult` is a function that we can call. How about `testResult`? And use `const {test} = testResult` a few lines below 👀